tests/test_services.py

```diff
@@ -169,6 +169,18 @@ def test_update_service(self):
         # tearDown
         service.edit(name=name_before, description=description_before, version=version_before)
 
+    # test added in 3.0
+    @skip
```

Comment:
service have 'ref' field included in the responses. Did you activate these in the EXTRA_PARAMS?

Reply:
I will add a ticket to production to ensure that you can also 'filter' on ref.

Reply:
The ref is now added as filter to the `/api/services?ref=....` and deployed to `pim3-test`

Reply:
Yes, they are activated, but they need to be in production